src/object/schema_manager.c

```diff
@@ -2254,6 +2254,7 @@ sm_downcase_name (const char *name, char *buf, int buf_size)
   if (name == NULL || name[0] == '\0')
     {
       ERROR_SET_WARNING (error, ER_SM_INVALID_ARGUMENTS);
+      buf[0] = '\0';
```

Comment:
It is recommended to move the inspection of the 2261 line of assert() to the front of this if block.

Reply:
Good point, it needs to move.

Reply:
Thank you. I will change it.